### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by dimonlex**
A few editorial changes

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Comment by dimonlex** [comment]
editorial

File: README.md [hunk 1]
```diff
@@ -75,4 +76,4 @@ Go to [contribution document](CONTRIBUTING.md) to read more how can you help us
 
 # Feedback
 
-Whether it helps you or not - we'd LOVE to hear from you.  Please let us know what you think and of course, how we can make it better.
+Whether it helps you or not - we'd LOVE to hear from you.  Please let us know what you think and of course, how we can make it better.  Please join our growing community on Discord or here.
```
Comment (jasmingacic) [inline README.md:79]: Where is discord link?

Comment (jasmingacic) [reply README.md:79]:
```suggestion
Whether it helps you or not - we'd LOVE to hear from you.  Please let us know what you think and of course, how we can make it better.  Please join our growing community on Discord or [here](https://discord.com/invite/6zupCZFQbe).
```

**Comment by bioslikk** [comment]
Added these changes to https://github.com/kubeshop/testkube/pull/855 and merget it.
We can close this PR now.

**Pull request closed without merging**